src/olympia/applications/templates/applications/appversions.html

```diff
@@ -32,10 +33,11 @@
 </div>
 {% endfor %}
 
-<p>{% trans url=_('http://developer.mozilla.org/en/docs/Install_Manifests') %}
+<p>{% trans amo_manifests_mdn_link_open='<a href="http://developer.mozilla.org/en/docs/Install_Manifests">'|safe,
+  amo_manifests_mdn_link_close='</a>'|safe%}
```

Comment:
Please add a whitespace before `%}` so that it's easier to read and follows the style in our templates.